.github/workflows/appsignals-e2e-eks-test.yml

```diff
@@ -65,8 +64,10 @@ jobs:
           delete_log_group="aws logs delete-log-group --log-group-name '${{ env.LOG_GROUP_NAME }}' --region \$REGION"
           sed -i "s#$delete_log_group##g" clean-app-signals.sh
 
-      - name: Generate testing id
-        run: echo TESTING_ID="${{ inputs.aws-region }}-${{ github.run_id }}-${{ github.run_number }}" >> $GITHUB_ENV
+      - name: Generate testing id and sample app namespace
+        run: |
+          echo TESTING_ID="${{ inputs.aws-region }}-${{ github.run_id }}-${{ github.run_number }}" >> $GITHUB_ENV
+          echo SAMPLE_APP_NAMESPACE="sample-app-namespace-${{ github.run_id }}-${{ github.run_number }}" >> $GITHUB_ENV
```

Comment:
Change it to `ns-${{ github.run_id }}-${{ github.run_number }}`, less risk of the string getting too long